content/en/agent/proxy.md

```diff
@@ -506,6 +506,19 @@ backend datadog-network-devices-netflow
     # server mothership ndmflow-intake.{{< region-param key="dd_site" >}}:443 check port 443 ssl verify required ca-file <PATH_TO_CERTIFICATES>
 ```
 
+{{< site-region region="us,eu,us3,us5,ap1" >}}
+For the following configuration, replace `<site_url>` with {{< region-param key="dd_site" >}}:
```

Comment:
Is there a reason why we don't include these snippets in the main config section? I think they should be fine included in the current sections already.

Same question for the other changes in this file.

Reply:
Per slack conversation , I moved these snippets into the main config section(s), with a note:
#remove this section if your environment does not support Remote Configuration